libmscore/figuredbass.cpp

```diff
@@ -1749,7 +1749,8 @@ void FiguredBass::writeMusicXML(XmlWriter& xml, bool isOriginalFigure, int crEnd
 FiguredBass* Score::addFiguredBass()
       {
       Element* el = selection().element();
-      if (el == 0 || (el->type() != ElementType::NOTE && el->type() != ElementType::FIGURED_BASS)) {
+      if (el == 0 || (el->type() != ElementType::NOTE && el->type() != ElementType::REST
```

Comment:
`!el` instead of `el == 0`.

Reply:
Maybe, but this is just changing existing code.